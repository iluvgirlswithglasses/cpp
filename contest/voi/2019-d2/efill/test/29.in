39
-80057866 -22842534
-80057142 -51823942
79552898 -80015640
-80015058 78801964
79930416 -79807088
-79890546 -79631466
-79307106 79974486
80019230 77853688
79975122 79394584
77953572 80029252
-47778682 80057716
77233256 80032492
80046626 68230418
-80053820 75731760
-80057766 64776158
80051662 -78693942
-79582702 -79892442
80055946 -60413428
-63910338 80048554
-79448346 -79958286
-78149910 80037878
-73850766 -80055658
-79901186 79932912
79465724 79906542
34098340 -80058044
-11385072 -80057534
79704790 79791170
80054652 28485076
71426580 -80054416
-78363092 -80045432
-80039594 -78318376
55775272 80054728
80033604 76724046
79806494 -79988526
80056054 -58767970
80052728 64189048
80056378 -52110014
30052664 80058024
79353014 79936450
-51076458 -22841810 -51075734 -51823218
