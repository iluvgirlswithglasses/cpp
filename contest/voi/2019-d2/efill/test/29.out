601521661.47
